docs/tools/feature-flags/how-to-use-feature-flags.md

```diff
@@ -0,0 +1,113 @@
+# How to use Feature Flags
+
+## Introduction
+
+MilMove uses [Flipt](https://www.flipt.io) for feature flags. See [ADR 0082 Use Flipt for Feature Flags](../../adrs/0082-use-flipt-feature-flags.md) for why Flipt was chosen and [ADR 0084 Deploy Flipt using ECS service discovery](../../adrs/0084-deploy-flipt-service-discovery.md) for how Flipt is deployed in AWS.
+
+While MilMove uses Flipt, it has a generic feature flag API that hides most of Flipt's implementation details. This allows for using environment variables to simulate feature flags in local development without running a Flipt server and would facilitate moving to another feature flag provider in the future should that be desirable.
+
+The feature flag data is stored in a separate, private repository `transcom/milmove-feature-flags`.
+
+As for 2023-09-06, feature flags have not been used for a full end to end feature deployment. This document may have some gaps because of that.
+
+### Feature Flag Evaluation
+
+First, familiarize yourself with the key [Flipt Concepts](https://www.flipt.io/docs/concepts).
+
+Each deployed environment (demo, exp, loadtest, stg, prd) is in its
+own [Flipt Namespace](https://www.flipt.io/docs/concepts#namespaces) and only that single namespace is available in each environment.
+
+Flipt supports two kinds of feature flags:
+ * [Boolean Flags](https://www.flipt.io/docs/concepts#boolean-flags)
+ * [Variant Flags](https://www.flipt.io/docs/concepts#variant-flags)
+
+Boolean flags are for when a feature can only be turned on or off. Variant flags are for when a feature might have multiple options.
+
+The most common [Entity](https://www.flipt.io/docs/concepts#entities) to work with is a user and that is probably how most feature flags will be used: to determine if a particular user should have access to a feature or not.
+
+The [Context](https://www.flipt.io/docs/concepts#context) contains multiple bits of information that can by used by flipt to determine if a feature should be available to a user or not.
+
+### MilMove Backend Feature Flag Service
+
+A [backend service object](../../backend/guides/service-objects/overview.md) has been created for feature flags that hides the specific Flipt API. This allows for using environment variable based feature flags in local development and/or when flipt is not available.
```

Comment:
```suggestion
We use the [backend service object](../../backend/guides/service-objects/overview.md) for feature flags to wrap the specific endpoints in the Flipt API. This allows for using environment variable based feature flags in local development and/or when Flipt is not available.
```

Reply:
Moving this out of the passive and into the active voice by deliberately calling out that we have service objects for the Flipt API in the server-side code to avoid using the Flipt API endpoint.